### PR TITLE
Dread: Final Artaria revision

### PR DESCRIPTION
**Pull request by XenoWars**
This PR finishes off the Artaria logic revision.

- Added pseudo-wave to the Water Reservoir Blob
- Various methods of climbing Waterfall
- Wave Beam for the rotatable in Waterfall
- Removed the powerbeam tile nodes in Chain Reaction Room as they are redundant post-reaction
- Removed Speed tile node in the room immediately following
- Added Climb Sloped Tunnels to get back into Chain Reaction Room
- Revised a bit of Teleport to Cataris
- Added some options to reach the item in EMMI Zone First Entrance

closes #3661

### CODE REVIEW

File: randovania/games/dread/json_data/Artaria.txt [hunk 1]
```diff
@@ -593,7 +593,12 @@ Extra - asset_id: collision_camera_005
   > Door to EMMI Zone Entrance Hallway
       Can Slide
   > Next to Pickup
-      Grapple Beam or Spider Magnet or Space Jump or Simple IBJ
+      Any of the following:
+          Spider Magnet or Space Jump or Simple IBJ
+          Grapple Beam and Grapple Movement (Beginner)
+          Walljump (Beginner) and Use Spin Boost
+          Flash Shift and Walljump (Intermediate)
+          Morph Ball and Single-wall Wall Jump (Intermediate)
```
Comment (Hugoshido) [inline randovania/games/dread/json_data/Artaria.txt:601]: Couldn´t we add Pseudo Wave (Beginner) + Legde jump (Beginner)?

File: randovania/games/dread/json_data/Artaria.txt [hunk 2]
```diff
@@ -1288,10 +1295,9 @@ Extra - asset_id: collision_camera_011
               All of the following:
                   Gravity Suit
                   Simple IBJ or Use Spin Boost
+              Movement (Intermediate) and Lay Cross Comb
```
Comment (Hugoshido) [inline randovania/games/dread/json_data/Artaria.txt:1298]: Could add PB + Speed Booster

File: randovania/games/dread/json_data/Artaria.txt [hunk 3]
```diff
@@ -1288,10 +1295,9 @@ Extra - asset_id: collision_camera_011
...
   > Tile Group (BOMB) 3
-      Any of the following:
-          Space Jump or Infinite Bomb Jump (Intermediate)
-          Gravity Suit and Simple IBJ
+      Space Jump or Simple IBJ
```
Comment (Hugoshido) [inline randovania/games/dread/json_data/Artaria.txt:1300]: Could add PB + Speed Booster

File: randovania/games/dread/json_data/Artaria.txt [hunk 4]
```diff
@@ -1366,11 +1372,17 @@ Extra - asset_id: collision_camera_011
...
+          Any of the following:
+              Wave Beam
+              Slide and Pseudo-Wave Beam (Beginner) and Slide Jump (Beginner)
   > Door to Varia Suit Tutorial South
       All of the following:
           After Artaria - Thermal Device
-          Space Jump or Simple IBJ
+          Any of the following:
+              Space Jump or Simple IBJ
+              Walljump (Intermediate) and Use Spin Boost
```
Comment (Hugoshido) [inline randovania/games/dread/json_data/Artaria.txt:1385]: Gravity + Speed Boster also possible here

File: randovania/games/dread/json_data/Artaria.txt [hunk 5]
```diff
@@ -3754,11 +3721,15 @@ Extra - asset_id: collision_camera_056
...
   > Right of Rotatable
       All of the following:
-          Gravity Suit
-          Space Jump or Simple IBJ
+          Any of the following:
+              Gravity Suit
+              After Artaria - Water Reservoir Blob and Perform WBJ
+          Any of the following:
+              Simple IBJ or Use Spin Boost
+              Flash Shift and Walljump (Intermediate)
```
Comment (Hugoshido) [inline randovania/games/dread/json_data/Artaria.txt:3732]: Can add Speed Boost here

**Review by Hugoshido — COMMENTED** [review]
Just some additions

File: randovania/games/dread/json_data/Artaria.txt (hunk 1 above)
Comment (XenoWars) [reply randovania/games/dread/json_data/Artaria.txt:601]: Unsure what a pseudo or a ledge jump (which is not even a trick) would do here. You're climbing the walls up, not shooting anything.

File: randovania/games/dread/json_data/Artaria.txt (hunk 2 above)
Comment (XenoWars) [reply randovania/games/dread/json_data/Artaria.txt:1298]: Not PBs, but you can do speedbooster yeah.

File: randovania/games/dread/json_data/Artaria.txt (hunk 3 above)
Comment (XenoWars) [reply randovania/games/dread/json_data/Artaria.txt:1300]: Just speed. PBs would not be considered cause it's covered by the tile block node weakness, and the blocks could be randomized to SA or Missile which would mean PBs would just be wasted.

File: randovania/games/dread/json_data/Artaria.txt (hunk 4 above)
Comment (XenoWars) [reply randovania/games/dread/json_data/Artaria.txt:1385]: Can you actually get up to the energy recharge with the speed still?

File: randovania/games/dread/json_data/Artaria.txt (hunk 5 above)
Comment (XenoWars) [reply randovania/games/dread/json_data/Artaria.txt:3732]: This is true, but it's more than just that due to door rando

File: randovania/games/dread/json_data/Artaria.txt (hunk 1 above)
Comment (Hugoshido) [reply randovania/games/dread/json_data/Artaria.txt:601]: Coming from early cloak and jumping up there is what I meant with this

File: randovania/games/dread/json_data/Artaria.txt (hunk 4 above)
Comment (Hugoshido) [reply randovania/games/dread/json_data/Artaria.txt:1385]: Yeah it's enough I tested it.

Comment (XenoWars) [reply randovania/games/dread/json_data/Artaria.txt:1385]: how tight is it?

**Review by Hugoshido — APPROVED** [review]

**Pull request merged**